optimum/habana/transformers/models/llama/configuration_llama.py

```diff
@@ -50,8 +49,8 @@ def __init__(
             rope_scaling,
             attention_bias,
             attention_dropout,
+            mlp_bias,
             **kwargs,
         )
 
-        self.mlp_bias = mlp_bias
         self.fused_qkv = fused_qkv
```

Comment:
for most of our llama ci tests,. we see this repeated error 
```bash 
[rank0]:     self.self_attn = GaudiLlamaAttention(config=config, layer_idx=layer_idx)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/optimum/habana/transformers/models/llama/modeling_llama.py", line 367, in __init__
[rank0]:     if config.fused_qkv:
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/transformers/configuration_utils.py", line 264, in __getattribute__
[rank0]:     return super().__getattribute__(key)
[rank0]: AttributeError: 'LlamaConfig' object has no attribute 'fused_qkv'
```
I can't seem to see this arg in HF upstream: https://github.com/huggingface/transformers/blob/v4.43.0/src/transformers/models/llama/configuration_llama.py#L165-L166
but maybe it needs to be set somewhere elsewhere?

Reply:
Test failure is from newly added: `tests/test_trainer.py::GaudiTrainerIntegrationTest::test_multiple_peft_adapters - AttributeError: 'LlamaConfig' object has no attribute 'fused_qkv'`
 
Potential rootcause: The code for fused_qkv, was added here: https://github.com/huggingface/optimum-habana/pull/1102 and the expectation is for the model's config.json to be updated to enable it. So apparently the model's config.json may not have the attribute. 
The one we're using definitely doesn't: https://huggingface.co/trl-internal-testing/tiny-random-LlamaForCausalLM/blob/main/config.json. 
We could check if the config even has the attribute with hasattr.

Also are you missing any file for this test like: `tests/fixtures/sample_text.txt` ? That is the next error I see after using the hasattr checks.

Reply:
Arf yeah, `peft` was not part of the `tests` extra in `setup.py` so it skipped this new test and I didn't see that it fails.
It should be all good now for this test and for Llama, I just pushed a commit to fix this.